community/modules/compute/htcondor-execute-point/README.md

```diff
@@ -0,0 +1,75 @@
+## Description
```

Comment:
What is the medium term plan for documentation. Norm has been to include an example snip of the module as it would appear in a blueprint.

Reply:
I've put in a snippet. Once this PR is merged I will modify #419 to include an update to this file that links to the public example

Reply:
In interest of not forgetting, I added the link to the module on the assumption that #419 will be merged soon enough after this PR.

Done on `HEAD` of PR.